cherab/solps/formats/raw_simulation_files.py

```diff
@@ -168,6 +170,19 @@ def load_solps_from_raw_output(simulation_path, debug=False):
             total_radiation[1:-1, 1:-1] -= eradt_raw_data
             sim.total_radiation = total_radiation / mesh.vol
 
+        # Obtaining molecular and total H-alpha radiation
+        halpha_wavelength = OpenADAS().wavelength(hydrogen, 0, (3, 2))
```

Comment:
I think it would be better (correct and clearer) if this respected isotopes. I know that it will not change the radiated power much and is negligible, but since the issue with the isotope wavelength is being fixed ;-)

Reply:
I agree, however this might be tricky in the case of multi-isotope simulation. EIRENE sums H-alpha emission over all isotopes, therefore the effective Balmer-alpha photon energy must be calculated with respect to local isotope concentrations. I'll try to do this photon to energy conversion more accurately.